x-pack/test/security_solution_api_integration/config/ess/config.base.ts

```diff
@@ -62,7 +62,7 @@ export function createTestConfig(options: CreateTestConfigOptions, testFiles?: s
         ...xPackApiIntegrationTestsConfig.get('esTestCluster'),
         license,
         ssl,
-        serverArgs: [`xpack.license.self_generated.type=${license}`],
```

Comment:
Is this an intended change or just something you added while testing?

Reply:
Nope 😨